src/profiles/registry.json

```diff
@@ -0,0 +1,28 @@
+{
+    "genericProfiles" : [
+        "button-controller", 
+        "thumbstick-controller", 
+        "touchpad-controller", 
+        "thumbstick-touchpad-controller"
+    ],
+    "customProfiles" : {
+        "google-daydream" : [ "touchpad-controller" ],
+        "htc-vive" : [ "touchpad-controller" ],
+        "microsoft-045e-065d" : [ "thumbstick-touchpad-controller" ],
```

Comment:
This happens to be the "Samsung Odyssey", which could be exposed with a human readable name of "samsung-odyssey".

Reply:
During the WebVR timeframe, these were the strings that Microsoft chose to expose.  @thetuvix needs to weigh in on what Microsoft actually wants here.